### PR TITLE
fix: star model issue

### PR DESCRIPTION
**Pull request by konstantina-gkp**
Enhancement on Directory App.

### CODE REVIEW

File: apps/directory/src/components/report-components/FactsTable.vue [hunk 1]
```diff
@@ -363,7 +363,7 @@ export default {
           next = next[trailPart];
         }
       }
-      return value ?? "Unknown";
+     return value ?? "Unknown";
```
Comment (chinook25) [inline apps/directory/src/components/report-components/FactsTable.vue:366]: Don't forget to run the code formatter :)

Comment (mswertz) [reply apps/directory/src/components/report-components/FactsTable.vue:366]: You can enable it to run automatic (but might not work on Windows) as described here https://molgenis.github.io/molgenis-emx2/#/molgenis/dev_quickstart?id=pre-commit-hook

**Review by mswertz — APPROVED** [review]
Looks good. However, for directory @esthervanenckevort should be always added as a reviewer (next to a developer) and when she approves you can merge.

File: apps/directory/src/components/report-components/FactsTable.vue (hunk 1 above)
Comment (chinook25) [reply apps/directory/src/components/report-components/FactsTable.vue:366]: just running 'gradle build' or 'gradle format' will also run the formatter :)

**Pull request merged**